lib/releases.js

```diff
@@ -76,38 +76,50 @@ const contributorsSentence = ({ commits, pullRequests }) => {
   }
 }
 
-const categorizePullRequests = (pullRequests, config) => {
-  const { 
-    'exclude-labels': excludeLabels,
-    'include-labels': includeLabels,
-    categories 
-  } = config
-  const allCategoryLabels = categories.flatMap(category => category.labels)
-  const uncategorizedPullRequests = []
-  const categorizedPullRequests = [...categories].map((category) => {
-    return { ...category, pullRequests: [] }
-  })
-
-  const filterExcludedPullRequests = (pullRequest) => {
+const getFilterExcludedPullRequests = (excludeLabels) => {
+  return (pullRequest) => {
     const labels = pullRequest.labels.nodes
-
     if (labels.some((label) => excludeLabels.includes(label.name))) {
       return false
     }
     return true
   }
+}
 
-  const filterIncludedPullRequests = pullRequest => {
+const getFilterIncludedPullRequests = (includeLabels) => {
+  return (pullRequest) => {
     const labels = pullRequest.labels.nodes
-
-    if (includeLabels.length == 0 ||
-        labels.some(label => includeLabels.includes(label.name))) {
+    if (
+      includeLabels.length == 0 ||
+      labels.some((label) => includeLabels.includes(label.name))
+    ) {
       return true
     }
     return false
   }
+}
 
-  const filterUncategorizedPullRequests = pullRequest => {
+const categorizePullRequests = (pullRequests, config) => {
+  const {
+    'exclude-labels': excludeLabels,
+    'include-labels': includeLabels,
+    categories,
+  } = config
+  const allCategoryLabels = categories.flatMap((category) => category.labels)
+  const uncategorizedPullRequests = []
+  const categorizedPullRequests = [...categories].map((category) => {
+    return { ...category, pullRequests: [] }
+  })
+
+  const filterExcludedPullRequests = getFilterExcludedPullRequests(
+    excludeLabels
+  )
+
+  const filterIncludedPullRequests = getFilterIncludedPullRequests(
+    includeLabels
+  )
```

Comment:
This reads funny...

Could we not call getFilterIncludedPullRequests() directly in the filtering below on line [137-138](https://github.com/darkdreamingdan/release-drafter/blob/781e89dd260b0fdd18bfddb540005c710ccab74f/lib/releases.js#L137-L138)?

Reply:
Wait why was getFilterExcludedPullRequests and getFilterIncludedPullRequests added when they are not used else.

I had them enclosed because nobody else used them...

Reply:
> This reads funny...
> 
> Could we not call getFilterIncludedPullRequests() directly in the filtering below on line [137-138](https://github.com/darkdreamingdan/release-drafter/blob/781e89dd260b0fdd18bfddb540005c710ccab74f/lib/releases.js#L137-L138)?

Yes, this is possible.  I just wasn't making significant changes to this function for this PR, and this is superficial.

> Wait why was `getFilterExcludedPullRequests` and `getFilterIncludedPullRequests` added when they are not used else.
> 
> I had them enclosed because nobody else used them...

They are used.  They are re-used in the version-resolver filtering, which is why i've pulled them out.

Reply:
Right than proceed with removing the const for `filterExcludedPullRequests` and `filterIncludedPullRequests`.

Reply:
Done